src/components/SendPopOver/index.jsx

```
@@ -18,27 +21,35 @@ const SendPopOver = () => {
     router.push('/home/send/custom-derivation');
   }, [router]);
 
+  const handleCustomDerivation = useCallback(() => {
+    popoverRef.current?.close();
+    if (!isCheckedStored) {
+      setShowCustomDerivationModal(true);
+    } else {
+      setShowConfirmModal(true);
+    }
+  }, [])
```

ESLint

[error] React Hook useCallback has a missing dependency: 'isCheckedStored'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)